commands/generate.js

```diff
@@ -46,7 +46,7 @@ trim_trailing_whitespace = false
 		if (err) {
 			fs.writeFile(configFile, ret, err => {
 				if (err) {
-					atom.notifications.addError(err);
+					atom.notifications.addError(err.message);
```

Comment:
Can you add the `err.stack` as the `detail` option?

What do you think about opening an issue on Atom for `addError` and `addFatal` accepting an Error object? That would be a nice shorthand for displaying errors, and I bet most people forget to include the stack, like here.